_includes/contact_form.html

```diff
@@ -57,7 +57,7 @@
         ),
         new TopicEntry(
           'Bug',
-          "{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.bug_report }}",
+          "{{ site.data.[page.lang].settings.contact_page.support_request_form.subtopics.blank_screen }}",
```

Comment:
Looks like `blank_screen` is missing from English data. Is it supposed to replace `bug_report` ?